app/src/instrumentation/initExportFunctions.ts

```diff
@@ -5,46 +5,50 @@ import {
 import { prismaClientForRawQueries } from 'src/prisma-client'
 
 // specify license and attribution for data export
-const license = "'ODbL 1.0, https://opendatacommons.org/licenses/odbl/'"
-const attribution = "'OpenStreetMap, https://www.openstreetmap.org/copyright; Radverkehrsatlas.de'"
+// const license = "'ODbL 1.0, https://opendatacommons.org/licenses/odbl/'"
+// const attribution = "'OpenStreetMap, https://www.openstreetmap.org/copyright; Radverkehrsatlas.de'"
 
 export async function initExportFunctions(tables: typeof exportApiIdentifier) {
   return Promise.all(
-    tables.map((tableName) => {
+    tables.map(async (tableName) => {
       const functionName = exportFunctionIdentifier(tableName)
+      const tagKeyQuery: Array<{ key: string }> = await prismaClientForRawQueries.$queryRawUnsafe(`
+        SELECT DISTINCT jsonb_object_keys(tags) AS key
+        FROM public."${tableName}"
+      `)
+      const metaKeyQuery: Array<{ key: string }> = await prismaClientForRawQueries.$queryRawUnsafe(`
+        SELECT DISTINCT jsonb_object_keys(meta) AS key
+        FROM public."${tableName}"
+      `)
+      const tagKeys = tagKeyQuery.map(({ key }) => `tags->>'${key}' as "${key}"`).join(',')
+      const metaKeys = metaKeyQuery.map(({ key }) => `meta->>'${key}' as "${key}"`).join(',')
 
       return prismaClientForRawQueries.$transaction([
-        prismaClientForRawQueries.$executeRaw`SET search_path TO public;`,
+        prismaClientForRawQueries.$executeRaw`SET search_path public;`,
```

Comment:
Vielleicht obsolet, da wir das "public" durch den anderen PR gelöscht haben; aber dass hier kein "TO" steht, kommt mir komisch vor; unten steht es noch.